Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -178,6 +177,7 @@ struct FPTIBatchData: Codable {
 
         let platform = "iOS"
 
+        /// This value should be the shopper session ID returned from your server SDK request
```

Comment:
Since we also use this for other modules that aren't shopper insights maybe something like:
```suggestion
        /// Either a randomly generated session ID or the shopper session ID passed in by a merchant
```